internal/controllers/topology/cluster/patches/engine_test.go

```diff
@@ -191,7 +191,7 @@ func TestApply(t *testing.T) {
 								{
 									Op:    "add",
 									Path:  "/spec/template/spec/resource",
-									Value: &apiextensionsv1.JSON{Raw: []byte(`"default-mp-worker-infra"`)},
+									Value: &apiextensionsv1.JSON{Raw: []byte(`"defaultMPWorkerVariable"`)},
```

Comment:
More or less. I was trying to make this test case more easily understandable. So instead of adding only new variables that are better and then inconsistent with the existing ones I also renamed these

Reply:
(and the changes below also affect this test case because we use the same Cluster / ClusterClass for all test cases)

Reply:
You're absolutely right. This shouldn't have been changed. Fixed, PTAL